kubetest/main.go

```diff
@@ -140,6 +142,8 @@ func defineFlags() *options {
 	flag.StringVar(&o.stage.dockerRegistry, "registry", "", "Push images to the specified docker registry (e.g. gcr.io/a-test-project)")
 	flag.StringVar(&o.save, "save", "", "Save credentials to gs:// path on --up if set (or load from there if not --up)")
 	flag.BoolVar(&o.skew, "skew", false, "If true, run tests in another version at ../kubernetes/hack/e2e.go")
+	flag.BoolVar(&o.soak, "soak", false, "If jobs runs as soak mode")
```

Comment:
Nit: "If true, job runs in soak mode" is probably more consistent with the other usage strings.